protocol/requests/mute_community_request.go

```diff
@@ -0,0 +1,26 @@
+package requests
+
+import (
+	"errors"
+
+	"github.com/status-im/status-go/eth-node/types"
+)
+
+var ErrInvalidMuteCommunityParams = errors.New("mute-community: invalid params")
+
+type MuteCommunity struct {
+	CommunityID types.HexBytes `json:"id"`
+	MutedType   MutingVariation
```

Comment:
Is the missing JSON tag on `MutedType` intentional? If not, it would be better if we added `json:"mutedType"`